docs/basic-docs/atomicdex-api-20/enable_eth_with_tokens.md

```diff
@@ -0,0 +1,244 @@
+| ticker                                       | string                                                      | Ticker of the platform protocol coin. Options: `ETH`, `AVAX`, `BNB`, `FTM`, `MATIC`, `ONE`, `ETH-ARB20`                                                                                                                                                                                                                                         |
+| gas_station_url                              | string (optional for ETH/ERC20 and other gas model chains)  | url of [ETH gas station API](https://docs.ethgasstation.info/); The AtomicDEX API uses [eth_gasPrice RPC API](https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gasprice) by default; when this parameter is set, the AtomicDEX API will request the current gas price from Station for new transactions, and this often results in lower fees |
+| gas_station_decimals                         | integer (optional for ETH/ERC20 and other gas model chains) | Defines the decimals used to denominate the gas station response to gwei units. For example, the ETH gas station uses 8 decimals, which means that "average": 860 is equal to 86 gwei. While the Matic gas station uses 9 decimals, so 860 would mean 860 gwei exactly. Defaults to `8`                                                         |
+| gas_station_policy.policy                    | string (optional for ETH/ERC20 and other gas model chains)  | Defines the method of gas price calculation from the station response. `"MeanAverageFast"` will use the mean between average and fast fields. `"Average"` will return a simple average value. Defaults to `"MeanAverageFast"`.                                                                                                                  |
+| mm2                                          | integer                                                     | Required if not set in `coins` file. Informs the AtomicDEX API whether or not the coin is expected to function. Accepted values are `0` or `1`                                                                                                                                                                                                  |
+| priv_key_policy                              | string (optional)                                           | defaults to `ContextPrivKey`, value can be `Metamask` when the AtomicDEX-API is built targeting `wasm`                                                                                                                                                                                                                                          |
```

Comment:
I dont think metamask is active in dev yet. @sergeyboyko0791 can you please list the current usable options for `priv_key_policy` in dev branch?

Reply:
In native `priv_key_policy` can only be either `ContextPrivKey` or `Trezor`.
Only in WASM and only for ETH it can also be `Metamask`, it's merged into dev, so I think we can mention it in the docs already despite Metamask is still in progress

Reply:
does this commit make it better? [`ccb1b60` (#420)](https://github.com/KomodoPlatform/developer-docs/pull/420/commits/ccb1b60f0c28567cd98a37f2b9aa12ab26649346)